script/deploy

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+## Validations
+[ -z "${DOCKER_USER}" ] && { echo "Need to set DOCKER_USER"; exit 0; }
```

Comment:
I get you, but as we're going CD here do we not want the build to fail if deploy fails?

Or at least have a way of getting clear visibility of that, so we avoid thinking somethings deployed when it hasn't?